test/functional/test_cli_commands.py

```diff
@@ -463,6 +463,12 @@ def tearDown(self):
         node1.stop()
         node1.terminate_workspace()
         self.node_info = None
+        fake_config_2 = fixtures.FakeConfig()
+        node_info_2 = fake_config_2.get_node_info()
```

Comment:
@WinnieXU1 the node might be left is named `test2`, to destroy this node, you need `node_info_2['name']="test2"` here